src/auth/auth.ts

```diff
@@ -121,6 +124,20 @@ export class BaseAuth<T extends AbstractAuthRequestHandler> implements BaseAuthI
       .then((decodedIdToken: DecodedIdToken) => {
         // Whether to check if the token was revoked.
         if (checkRevoked || isEmulator) {
+          // Whether user has been disabled.
+          if (checkRevoked) {
+            const uid = decodedIdToken.uid;
+            return this.getUser(uid)
```

Comment:
You are calling `getUser` twice. Here and in `verifyDecodedJWTNotRevoked`. You need to refactor `verifyDecodedJWTNotRevoked` to do both checks.

See here: https://github.com/firebase/firebase-admin-node/blob/b88f8919c67c252b31d6b4b455fd7ba9b6226131/src/auth/auth.ts#L738